app/views/name/_form_name.html.erb

```diff
@@ -10,14 +10,27 @@
 <% end %>
 
 <% if !@name.locked || in_admin_mode? %>
+  <div class="form-inline push-down">
+    <div class="form-group">
+      <label for="name_icn_id"><%= :form_names_icn_id.t %>: </label>
+      <%= form.text_field(:icn_id, class: "form-control", size: 8) %>
+      <%= :form_names_identifier_help.t %>
+    </div>
+  </div>
+
   <div class="form-inline push-down">
     <div class="form-group">
       <label for="name_rank"><%= :Rank.t %>:</label>
-      <%= form.select(:rank, Name.all_ranks.map {|r| [rank_as_string(r), r]}, {}, {class: "form-control"}) %>
+      <%= form.select(
+            :rank, Name.all_ranks.map {|r| [rank_as_string(r), r]}, {},
+            {class: "form-control"}) %>
     </div>
+
     <div class="form-group indent">
       <label for="name_deprecated"><%= :Status.t %>:</label>
-      <%= form.select(:deprecated, [[:ACCEPTED.t, false], [:Deprecated.t, true]],{},{class: "form-control"}) %>
+      <%= form.select(
+            :deprecated, [[:ACCEPTED.t, false], [:Deprecated.t, true]],{},{
```

Comment:
Not that it makes a difference in this case sense all translations of "accepted" and "deprecated" are probably single words, but... it's very strange that it would request all caps for "accepted" but not for "deprecated"!  We should choose one or the other and be consistent.  (Probably my fault, but since you're fixing so many preexisting stylistic issues already, might as well nail this one, too!)

Reply:
I agree. I will make them consistent.